distribution/install.sh

```diff
@@ -207,6 +207,29 @@ add_to_path() {
 # be updated if necessary.
 # ------------------------------------------------------------------------------
 
+get_gnu_musl_glibc() {
+  need_cmd ldd
+  need_cmd bc
+  need_cmd awk
+  # Detect both gnu and musl
+  # Also detect glibc versions older than 2.18 and return musl for these
+  local _ldd_version
+  local _glibc_version
+  _ldd_version=$(ldd --version)
+  if [[ $_ldd_version =~ "GNU" ]]; then
+    _glibc_version=$(echo "$_ldd_version" | awk '/ldd/{print $NF}')
+    if [ 1 -eq "$(echo "${_glibc_version} < 2.18" | bc)" ]; then
```

Comment:
Not that I expect we'll actually run into this, but this would falsely result in it returning true once glibc hits version `2.100`.

I see some bash semvar comparison implementations on: https://stackoverflow.com/questions/4023830/how-to-compare-two-strings-in-dot-separated-version-format-in-bash

Reply:
I want to hope this is fixed prior to 2.100 but let's take a look. :P

Reply:
I'm not going to change it for now - I think this works for the foreseeable future with less complexity than alternatives.

Reply:
:+1: yeah, that's fine. It seems unlikely we'll hit it before `cross` is patched or those old distros fall out of service.

Rather than a full semvar comparison, you could split the glibc version on `.` and compare major / minor independently with `bc`.

Or add a timebomb that would fail the script if `glibc_version` is longer than 4 characters (indicating that it hit `2.100`.